db/migrate/20220920143924_create_offenses.rb

```diff
@@ -0,0 +1,15 @@
+class CreateOffenses < ActiveRecord::Migration[6.0]
+  def change
+    create_table :tulsa_blotter_offenses do |t|
+      t.references :tulsa_blotter_arrests, null: true, foreign_key: true
+      t.string :description
+      t.string :case_number
+      t.string :court_date
+      t.string :bond_type
+      t.string :bound_amount
```

Comment:
Not sure if there's a reason to store `bond_amount` as a float instead of string? It's usually in $0.00 format on the website.